samples/meetings-live-caption/nodejs/index.js

```diff
@@ -28,9 +23,19 @@ server.engine('html', require('ejs').renderFile);
 server.set('view engine', 'ejs');
 server.set('views', __dirname);
 
-const port = process.env.port || process.env.PORT || 3978;
-server.listen(port, () => 
-    console.log(`\n${server.name} listening to http://localhost:${port}`)
+// Specify the paths to your SSL certificate and private key
+const sslOptions = {
+    key: fs.readFileSync(process.env.SSL_KEY_FILE),
```

Comment:
are we giving certificate values in env file ?

Reply:
yes